azure-pipelines/end-to-end-tests-dir/ci-verify-versions.ps1

```diff
@@ -2,14 +2,12 @@
 
 Copy-Item -Recurse "$PSScriptRoot/../e2e-assets/ci-verify-versions-registry" "$TestingRoot/ci-verify-versions-registry"
 git -C "$TestingRoot/ci-verify-versions-registry" @gitConfigOptions init
+git -C "$TestingRoot/ci-verify-versions-registry" @gitConfigOptions add --chmod=+x 'ports/executable-bit/some-script.sh'
```

Comment:
This additional test was added with the change to reuse the existing index done in response to `x-ci-verify-versions` failing on the real github.com/microsoft/vcpkg. Because I was always making new trees that don't have the git-only-on-Windows-anyway `+x` bit, the new trees had different SHAs than our version database says, resulting in failures.

In the following message to the team I sent when I just discovered this problem I'm using `git diff-tree` on some of the openssl trees and `ports\openssl\unix\configure` shows up because it's got a `+x`.

![image](https://github.com/user-attachments/assets/e146fc79-750a-4dd6-b65f-12c0de3af27b)